.eslintrc.js

```diff
@@ -29,7 +29,11 @@ module.exports = {
         'import/default': 'off',
         'import/order': 'error',
         'no-dupe-class-members': 'off',
+        // ts handles this check for us, also eslint does not include Bigint
+        'no-undef': 'off',
         'no-unused-vars': 'off',
+        // ts handles this check for us, also eslint does not include bigint
+        'valid-typeof': 'off',
```

Comment:
`valid-typeof` includes `bigint` starting in 6.0.0-rc.0

Reply:
yeah, we can revert the eslint changes if they no longer cause issues. Makes this PR a bit bigger than it needs to be. you wanna do that?